modules/core/core-export/src/main/java/com/enonic/xp/core/impl/export/writer/NodeExportPathResolver.java

```diff
@@ -1,14 +1,11 @@
 package com.enonic.xp.core.impl.export.writer;
 
-import java.nio.file.Path;
-import java.nio.file.Paths;
-
-import com.enonic.xp.node.NodePath;
-import com.enonic.xp.node.NodeVersionId;
-import com.enonic.xp.util.BinaryReference;
-
 public class NodeExportPathResolver
```

Comment:
Maybe make sense to rename this file, for instance to NodeExportPathConstants, because no any resolve method

Reply:
Will be part of some other cleanup. This PR is already overloaded